device_calendar/example/lib/presentation/event_item.dart

```diff
@@ -29,8 +29,8 @@ class EventItem extends StatelessWidget {
               child: new FlutterLogo(),
             ),
             new ListTile(
-                title: new Text(_calendarEvent.title),
-                subtitle: new Text(_calendarEvent.description)),
+                title: new Text(_calendarEvent.title ?? ''),
```

Comment:
Wouldn't this just work? As in, when null is provided then the text is basically empty?